src/toil_vg/vg_config.py

```diff
@@ -357,9 +356,8 @@
 # Use output store instead of toil for all intermediate files (use only for debugging)
 force-outstore: False
 
-# Toggle container support.  Valid values are Docker / Singularity / None
-# (commenting out or Null values equivalent to None)
-container: Docker
+# determine what container framework to use
```

Comment:
Why this change?  We lose information (what are valid inputs, and that containers can be disabled altogether).  And also, the code is checking for 'Docker' and not 'docker'.